vertical-pod-autoscaler/RELEASE.md

```diff
@@ -46,6 +47,45 @@ We use the issue to communicate what is state of the release.
 
 ## Build and stage images
 
+Select either the Automatic and Manual process below.
+
+### Option 1: (Preffered) Automatic
+
+NOTE: Currently this process can only be used for new minor releases. Patch
+releases need to follow the manual process below.
+
+Images are continuously built as part of the PR release process and are listed
+in the following repository:
+[gcr.io/k8s-staging-autoscaling](http://gcr.io/k8s-staging-autoscaling). Also
+see the
+[dashboard](https://testgrid.k8s.io/sig-autoscaling-vpa-images#post-autoscaler-push-vpa-images)
+for build status.
+
+To stage an image:
+
+1. Pick an image to promote from the latest automatically built images in the
+repository. Make note of the automatically generated tag (the tag will contain
+the date, the name of the latest tag in the repository and a commit hash). For
+example `v20250430-cluster-autoscaler-chart-9.46.6-81-g6a6a912b4`.
+
+2. Set the `BUILD_TAG` variable to this tag in your shell:
+
+```sh
+BUILD_TAG=<tag>
+```
+
+3. Now tag this image with the latest version tag:
+
+```sh
+cd vertical-pod-autoscaler/
+TAG=`grep 'const versionCore = ' common/version.go | cut -d '"' -f 2`
```

Comment:
I wonder if it makes sense to echo $BUILD_TAG and $TAG just to show the operator what's happening?

```suggestion
TAG=`grep 'const versionCore = ' common/version.go | cut -d '"' -f 2`
echo "Adding tag $TAG based on built tag $BUILD_TAG"
```

Reply:
Done, thanks!